pom.xml

```diff
@@ -44,7 +46,12 @@
 			<scope>provided</scope>
             <version>1.16.18</version>
 		</dependency>
-        <dependency>
+		<dependency>
```

Comment:
Why do you still need this?

Reply:
Because Spring default metrics doesn't support `Meter`. But maybe you are right and we should rely on Spring defaults here.

Reply:
The less dependencies the better. Gives your clients more freedom.